feat: sync with WCAG 2.2 (#34160)
mdn/content: files/en-us/web/accessibility/understanding_wcag/index.md

{{AccessibilitySidebar}}

This set of articles provides quick explanations to help you understand the steps that need to be taken to conform to the recommendations outlined in the {{glossary("WCAG", "Web Content Accessibility Guidelines")}} (WCAG) 2.2.

> The WCAG guidelines are a set of recommendations for making web content more accessible developed by W3C's {{Glossary("WAI")}}, primarily for people with disabilities — but also for all user agents, including highly limited devices, such as mobile phones.

Author: ```suggestion
The WCAG guidelines are a set of recommendations for making web content more accessible developed by W3C's {{Glossary("WAI")}}, primarily for people with disabilities — but also for all user agents, including some highly limited devices or services, such as digital assistants.
```

also let's bring the changes to the glossary term. (the first paragraph was kinda copied from the glossary one)

> 

Author: ```suggestion

WCAG 2.2 is the most recent version of the guidelines with WCAG 2.1 and 2.0 also widely used. WCAG 3.0 is the working draft.
Newer versions of WCAG are not meant to supersede previous versions, but WAI recommends using the most recent version to ensure the best possible accessibility for your website.

```

## The four principles

WCAG is broadly broken down into four principles — major things that web content **must be** to be considered accessible (see [Understanding the Four Principles of Accessibility](https://www.w3.org/WAI/WCAG22/Understanding/intro#understanding-the-four-principles-of-accessibility) for the WCAG definitions).

> Each of the links below will take you to pages that further expand on these areas, giving you practical advice on how to write your web content so it conforms to the success criteria outlined in each of the WCAG 2.0 and 2.1 guidelines that further sub-divides each principle.

Reviewer: ```suggestion
Each of the links below will take you to pages that further expand on these areas, giving you practical advice on how to write your web content so it conforms to the success criteria outlined in the WCAG 2.X four guiding principles.
```

Reviewer: we talk about 2.2 as the current standard, so let's make this generic.

## Which version should I use?

WCAG 2.2 is the most recent version of the guidelines and WCAG 3.0 is the working draft, but WCAG 2.1 and 2.0 is also widely used. WCAG is not meant to supersede previous versions, but WAI recommends using the most recent version of the guidelines to ensure the best possible accessibility for your website.

## Legal standing

This guide is intended to provide practical information to help you build better, more accessible websites. However, we are not lawyers, and none of this constitutes legal advice. If you are worried about the legal implications of web accessibility, we'd recommend that you check the specific legislation governing accessibility for the web/public resources in your country or locale, and seek the advice of a qualified lawyer.

[What is accessibility?](/en-US/docs/Learn/Accessibility/What_is_accessibility) and particularity the [Accessibility guidelines and the law](/en-US/docs/Learn/Accessibility/What_is_accessibility#accessibility_guidelines_and_the_law) section provide more related information.

## See also

> - [WCAG Overview](https://www.w3.org/WAI/standards-guidelines/wcag/)

Reviewer: ```suggestion
- [WCAG: Perceivable](https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Perceivable)
- [WCAG: Operable](/en-US/docs/Web/Accessibility/Understanding_WCAG/Operable)
- [WCAG: Understandable](/en-US/docs/Web/Accessibility/Understanding_WCAG/Understandable)
- [WCAG: Robust](/en-US/docs/Web/Accessibility/Understanding_WCAG/Robust)
- [WCAG text labels and names](/en-US/docs/Web/Accessibility/Understanding_WCAG/Text_labels_and_names)
- [WCAG keyboard accessiblity](/en-US/docs/Web/Accessibility/Understanding_WCAG/Keyboard)
- [WCAG Overview](https://www.w3.org/WAI/standards-guidelines/wcag/)
```
suggest adding all the guides
